Sources/TuistGenerator/Graph/Graph.swift

```diff
@@ -158,7 +158,7 @@ class Graph: Graphing {
     }
 
     var packages: [PackageNode] {
-        return Array(cache.packageNodes.values)
+        return cache.packages.values.flatMap { $0 }
```

Comment:
Wouldn't be more efficient to wrap it into an `Array`? Not that the users would have many packages as for this to be a bottleneck but if we can avoid this to add up, I'd do it.

Reply:
It is an array, i have to flatmap here to get a single array for all packages in the graph. The definition of the property is a dictionary whos value is an array.

The signature of the property has not changed